### PR TITLE
add searchmdn slash command

### PR DESCRIPTION
**Pull request by Asartea**
## Because
- We frequently link to MDN when answering questions, which usually requires the helper to google term, copy the MDN link, and then paste it.

## This PR
- Adds a command to search mdn via google, in the same way that the searchtop command works

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Callbacks command: Update verbiage`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [ ] If this PR adds new features or functionality, I have added new tests

### CODE REVIEW

File: new-era-commands/slash/searchmdn.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('searchmdn')
+    .setDescription('Search for text in MDN (with Google)')
+    .addStringOption((option) => option.setName('prompt').setDescription('term to search').setRequired(true))
+    .addUserOption((option) => option.setName('user').setDescription('user to ping')),
+  execute: async (interaction) => {
+    const userId = interaction.options.getUser('user')?.id;
+    const prompt = interaction.options.getString('prompt');
+    const searchUrl = `https://www.google.com/search?q=site:developer.mozilla.org+${prompt.replaceAll(' ', '+')}`;
```
Comment (Lofty-Brambles) [inline new-era-commands/slash/searchmdn.js:12]: Could we not use MDN's search function directly?
Also, using the `URLSearchParams` API would be appropriate here.

Comment (Asartea) [reply new-era-commands/slash/searchmdn.js:12]: I'm going to be honest; I have no memory of this PR

**Comment by wise-king-sullyman** [comment]
@Asartea based on your last comment are you not interested in continuing on this PR at this point?

**Comment by Asartea** [comment]
Yeah; while I still think it's a good idea, I'd probably just rewrite it from the ground up when I have time

**Pull request closed without merging**